library/globals.inc.php

```diff
@@ -2706,10 +2699,10 @@ function gblTimeZones()
         ),
 
         'fhir_base_url' => array(
-            xl('FHIR Server Base Address'),
+            xl('FHIR Server Base URI'),
```

Comment:
Just making sure you meant this one :)

Reply:
Started out to make obvious that can be used for new restful api as well but got distracted. Maybe one of the kids that seem to be always around, had candy:)